app/controllers/api/teams.php

```diff
@@ -790,27 +791,30 @@
 
         $memberships = array_filter($memberships, fn (Document $membership) => !empty($membership->getAttribute('userId')));
 
-        $memberships = array_map(function ($membership) use ($dbForProject, $team) {
-            $user = $dbForProject->getDocument('users', $membership->getAttribute('userId'));
-
-            $mfa = $user->getAttribute('mfa', false);
-            if ($mfa) {
-                $totp = TOTP::getAuthenticatorFromUser($user);
-                $totpEnabled = $totp && $totp->getAttribute('verified', false);
-                $emailEnabled = $user->getAttribute('email', false) && $user->getAttribute('emailVerification', false);
-                $phoneEnabled = $user->getAttribute('phone', false) && $user->getAttribute('phoneVerification', false);
+        $hideSensitiveFields = $project->getAttribute('auths', [])['teamHideSensitiveFields'] ?? false;
 
-                if (!$totpEnabled && !$emailEnabled && !$phoneEnabled) {
-                    $mfa = false;
+        $memberships = array_map(function ($membership) use ($dbForProject, $team, $hideSensitiveFields) {
+            $user = $dbForProject->getDocument('users', $membership->getAttribute('userId'));
+            $membership->setAttribute('teamName', $team->getAttribute('name'));
```

Comment:
The DB library has a a select query , so we should use that instead of fetching all attributes and filtering right? 🤔 
https://github.com/utopia-php/database/blob/main/src/Database/Query.php#L449-L452